### PR TITLE
Changed logo padding, title and other pages fonts

### PR DESCRIPTION
**Pull request by dgarciaperales**

### CODE REVIEW

File: src/styles/navstyles.css [hunk 1]
```diff
@@ -66,10 +104,15 @@
     background: var(--color-navyblue);
   }
   
-  .navbar-link {
-    text-decoration: none;
+  .navbar-link, .pages {
```
Comment (AmaraWilliams) [inline src/styles/navstyles.css:107]: When users hover over the navbar pages, they should be able to see it change color when it is hovered over. Because you wrapped the text around this css class instead of keeping it as part of the <Link> text, you need to add the styling for hovering here. In red is where my cursor was when I snapped this screenshot. As you can see, the text stayed white. Previously, it would change to a blue that is part of our color palette.
image

File: src/styles/navstyles.css [hunk 2]
```diff
@@ -16,11 +16,49 @@
...
+  }
+
+  nav {
+    display: flex;
+    align-items: center;
+  }
+
+  
+
+  h1, h2 {
```
Comment (AmaraWilliams) [inline src/styles/navstyles.css:57]: You already had styling for H2 on line 45 that gets overwritten by this code. Please remove the styling on H2 if you want to go with this styling.

File: src/styles/navstyles.css (hunk 1 above)
Comment (AmaraWilliams) [inline src/styles/navstyles.css:107]: Why the two names? Let's be consistent and just use the className "navbar-link".

File: src/styles/navstyles.css [hunk 3]
```diff
@@ -16,11 +16,49 @@
...
   body {
-    font-family: "Segoe UI Variable Display", "Segoe UI", -apple-system, Roboto,
-      sans-serif, serif;
+    font-family: "Roboto Slab", serif;
     margin: 0;
+    font-weight: bold;
     /* overflow-x: hidden; */
   }
+
+  header {
```
Comment (AmaraWilliams) [inline src/styles/navstyles.css:25]: Did you have any issues making the padding even on the left and right for the header? I think I had this issue last semester as well if so. Let me know if I can be of assistance.
image

File: src/styles/navstyles.css [hunk 4]
```diff
@@ -100,6 +143,37 @@
...
+  }
+
+  .nav-close-btn {
+    align-self: flex-end;
+    font-size: 1.5rem;
+    color: white;
+  }
+
+
   /*.navbar-grid-right {
```
Comment (AmaraWilliams) [inline src/styles/navstyles.css:177]: Please place any commented out or unused code into the [Knowledge Board](https://docs.google.com/document/d/1BXvILLekHlO0pEJhnBhob7JFHt3supxyvAGyhev482M/edit?usp=sharing) and summarize what it does and why it's been removed. You can then delete this code. Be sure to comment on your new code as well so we understand what the styling is trying to acheive for each class.

**Review by AmaraWilliams — CHANGES_REQUESTED** [review]
I love how it looks now, but there are issue within the codebase that need to be addressed before I'd merge. Also, be sure to comment properly on your added code and add any unused code and commented out code into the [knowledge board](https://docs.google.com/document/d/1BXvILLekHlO0pEJhnBhob7JFHt3supxyvAGyhev482M/edit?usp=sharing).

File: src/styles/navstyles.css
Comment (AmaraWilliams) [inline src/styles/navstyles.css:1]: There is a change in this file that indirectly changed the look of the footer making the text bolded. Make sure the styles you add do not interfere with other components that are not related to your branch.

Before (Master Branch):
image



After (your branch):
image

File: src/styles/navstyles.css (hunk 3 above)
Comment (joshrsamuel) [reply src/styles/navstyles.css:25]: Second this! The navigation buttons should be aligned to the right hand side of the nav bar

File: src/styles/navstyles.css (hunk 1 above)
Comment (joshrsamuel) [reply src/styles/navstyles.css:107]: Second this as well. It's important to have hover interactions to indicate clickability

File: src/components/newnav.js [hunk 5]
```diff
@@ -56,13 +62,21 @@ function Newnav({ transparent }) {
...
-                  <Link to={"/experiences"}>Experiences</Link>
-                  <Link to={"/about"}>About</Link>
-                  <Link to={"/contact"}>Contact</Link>
+                  <Link to={"/projects"}>
+                    <h3 className="pages">Projects</h3>
+                  </Link>
+                  <Link to={"/experiences"}>
+                    <h3 className="pages">Experiences</h3>
+                  </Link>
+                  <Link to={"/experiences"}>
```
[the remark this answers is not in the document]
Comment (joshrsamuel) [reply src/components/newnav.js:74]: Like Amara said, this needs to be fixed before we can push.

File: src/components/newnav.js
Comment (joshrsamuel) [inline src/components/newnav.js:1]: I also noticed some visual inconsistencies that were seemingly caused by this change. 

## Bottom of Experiences Page
image

## Vertical Position of "Empathy Bytes" Text on Home Page
image
The text is centered on the current site, but on your branch, the text is closer to the top of the page.

**Review by joshrsamuel — CHANGES_REQUESTED** [review]
